play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcHeader.scala.html

```diff
@@ -147,11 +174,6 @@
     @userResearchBanner.map { urb =>
         @hmrcUserResearchBanner(UserResearchBanner(language, urb.url))
     }
-    @if(displayHmrcBanner){
-      <div class="govuk-width-container">
-        @hmrcBanner(Banner(language, useTudorCrown))
-      </div>
-    }
```

Comment:
We need to double check if we should be changing the order of banners. Currently based on the prototype HMRC banner is going on top